packages/nimble-components/src/rich-text/viewer/styles.ts

```diff
@@ -25,6 +25,7 @@ export const styles = css`
         box-sizing: border-box;
         position: relative;
         color: inherit;
+        overflow-wrap: anywhere;
```

Comment:
Make sure to update the PR implementation description to match the changes

Reply:
Updated